packages/bot-web-ui/src/components/dashboard/tour-trigger-dialog.tsx

```diff
@@ -48,14 +44,13 @@ const TourTriggrerDialog = observer(() => {
     };
 
     const getTourHeaders = (tour_check: boolean, tab_id: number) => {
-        let text;
         if (!tour_check) {
-            if (tab_id === 1 && is_mobile) text = localize('Bot Builder guide');
-            else if (tab_id === 1) text = localize("Let's build a Bot!");
-            else text = localize('Get started on Deriv Bot');
-        } else if (tab_id === 1) text = localize('Congratulations');
-        else text = localize('Want to retake the tour?');
-        return text;
+            if (tab_id === 1) {
+                return is_mobile ? localize('Bot Builder guide') : localize("Let's build a Bot!");
+            }
+            return localize('Get started on Deriv Bot');
+        }
+        return tab_id === 1 ? localize('Congratulations') : localize('Want to retake the tour?');
```

Comment:
if tab_id is 1 then this line may not execute as line number 49 may return away from the function. Can you please check this?